src/customers/views/CustomerList/CustomerList.tsx

```diff
@@ -217,14 +219,14 @@ export const CustomerList: React.FC<CustomerListProps> = ({ params }) => {
         open={params.action === "save-search"}
         confirmButtonState="default"
         onClose={closeModal}
-        onSubmit={handleTabSave}
+        onSubmit={onPresetSave}
       />
       <DeleteFilterTabDialog
         open={params.action === "delete-search"}
         confirmButtonState="default"
         onClose={closeModal}
-        onSubmit={handleTabDelete}
-        tabName={maybe(() => tabs[currentTab - 1].name, "...")}
+        onSubmit={onPresetDelete}
+        tabName={presets?.[presetIdToDelete ?? 1 - 1]?.name ?? "..."}
```

Comment:
suggestion: maybe will be better to extract this to separate function like this:
```
  const getFilterPresetDeleteName = () => {
    if (!presetIdToDelete || !presets[presetIdToDelete - 1]) {
      return "...";
    }

    return presets[presetIdToDelete - 1].name;
  };
  ```

Reply:
I implemented it like this before - it was crashing in some edge cases